components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -160,199 +205,137 @@ + (NSDictionary *)catalogMetadata {
 #pragma mark - Example handlers
 
 - (void)showSingleLineStyleBanner {
-  if (self.bannerViewContainer) {
-    [self.bannerViewContainer removeFromSuperview];
+  if (self.bannerView) {
+    [self.bannerView removeFromSuperview];
   }
-  // Set up banner view container
-  UIView *bannerViewContainer = [[UIView alloc] initWithFrame:CGRectZero];
-  [self.view addSubview:bannerViewContainer];
-  bannerViewContainer.backgroundColor = [UIColor whiteColor];
-  self.bannerViewContainer = bannerViewContainer;
 
   MDCBannerView *bannerView = [[MDCBannerView alloc] init];
-  bannerView.image = [UIImage imageNamed:@"Email"];
+  bannerView.preferredContentWidth = exampleBannerContentWidth;
   bannerView.text = exampleShortText;
+  bannerView.backgroundColor = self.colorScheme.surfaceColor;
+  [self.view addSubview:bannerView];
+  self.bannerView = bannerView;
 
-  MDCButton *button = [bannerView.buttons firstObject];
+  MDCButton *button = [[MDCButton alloc] initWithFrame:CGRectZero];
   [button setTitle:@"DISMISS" forState:UIControlStateNormal];
-  UIFont *buttonFont = [MDCTypography body2Font];
-  [button setTitleFont:buttonFont forState:UIControlStateNormal];
-  [button setTitleFont:buttonFont forState:UIControlStateHighlighted];
   [button setTitleColor:self.colorScheme.primaryColor forState:UIControlStateNormal];
+  [button sizeToFit];
   button.backgroundColor = self.colorScheme.surfaceColor;
+  NSMutableArray *buttons = [NSMutableArray arrayWithObject:button];
+  bannerView.buttons = buttons;
```

Comment:
Tip: Consider using an inline literal array:

```objc
bannerView.buttons = @[ button ];
```

Reply:
The current API of `MDCBannerView` is `NSMutableArray<MDCButton *>`, which led to the code landed here (which I could change to `[@[button] mutableCopy]`. I will make myself note here to use the best/clean way to add buttons based a finalized API after API review if it sounds good to you.